origin-dapp/public/schemas/housing.json

```diff
@@ -3,6 +3,13 @@
   "type": "object",
   "required": ["name","category","description"],
   "properties": {
+    "listingType": {
+      "const": "fractional"
```

Comment:
Will this "take effect" or have any consequences if deployed to production before we "wire on" the calendar, etc?

I'm fairly certain that I should include my changes to the categories before this is merged. Several of these will be inaccurate or removed entirely.

Reply:
Good point @micahalcorn - I'll make a change so that this will still respect the `ENABLE_FRACTIONAL` environment variable, but we can have this in the schema without breaking anything. 

If your changes are ready to implement, I'm all for including them here. IMO, not a big deal to merge this and do them separately though if the final approval from J&M is going to take a while. If we were talking about origin-js schema changes that required a version bump, ofc I'd feel differently, but these are pretty low effort to change.

Reply:
I'll point a PR to here so that you can review before I hijack your PR.